linter-300.md

```diff
@@ -0,0 +1,139 @@
+
+We have also overhauled how linters are documented. Previously, all linters
+were documented on a single page and described in a quick blurb. This has
+grown unwieldy as lintr now exports 72 linters! Now, each linter gets its own
+page, which will make it easier to document any parameters, enumerate edge cases/
+known false positives, add links to external resources, etc.
+
+To make this even more navigable, we've also added some metadata to a database
+in the form of tags for each linter. For example, `brace_linter` has tags
+style, readability, default, and configurable. Each tag gets its own documentation
+page (e.g. `?readability_linters`) which describes the tag and lists all of the
+associated linters exported by lintr.
+
+To further facilitate discoverability, we've added `available_linters()` and
+`available_tags()` to list registered linters (including custom linters not
+exported by lintr itself; see `?available_linters`).
+
+Moreover, lintr's documentation is now available as a website thanks to
+Hadley Wickham's contribution to create a pkgdown website for the package:
+[lintr.r-lib.org](https://lintr.r-lib.org).
+
+## Google linters
+
+This release also features more than 30 new linters originally authored by Google developers.
+Google adheres mostly to the tidyverse style guide and uses lintr to improve the consistency
+of R code in its considerable internal R code base. These linters detect common issues with
+readability, consistency, and performance. Here are some examples:
+
+ - `any_is_na_linter` detects the usage of `any(is.na(x))`; `anyNA(x)` is nearly always a better choice,
+   both for performance and for readability
+ - `expect_named_linter` detects usage in [testthat](http://testthat.r-lib.org/) suites like
+   `expect_equal(names(x), c("a", "b", "c"))`; `testthat` also exports `expect_named()` which is
+   tailor made to make more readable tests like `expect_named(x, c("a", "b", "c"))`
+ - `vector_logic_linter` detects usage of vector logic operators `|` and `&` in situations where
+   scalar logic applies, e.g. `if (x | y) { ... }` should be `if (x || y) { ... }`. The latter
+   is more efficient and less error-prone.
+ - `strings_as_factors_linter` helps developers maintaining code that straddles the R 4.0.0 boundary,
+   where the default value of `stringsAsFactors`
+   [changed from `TRUE` to `FALSE`](https://developer.r-project.org/Blog/public/2020/02/16/stringsasfactors/),
+   by identifying usages of `data.frame()` that (1) have known string columns and (2) don't declare
+   a value for `stringsAsFactors`, and thus rely on the default.
+
+See the NEWS for the complete list.
+
+## Other improvements
+
+This is a big release -- almost 2 years in the making -- and there has been a plethora of smaller
+but nonetheless important changes to lintr. Please check the NEWS for a complete enumeration of these;
+here are a few more highlights:
+
+# What's next in lintr
```

Comment:
heading structure seems less than ideal.
Add a level 1 heading at the very top, or increase all levels down from here by one?

Reply:
maybe it was just missing an L1 header for "Release highlights"? added now